src/lsst/rsp/startup/services/labrunner.py

```diff
@@ -772,19 +706,14 @@ def _set_timeout_variables(self) -> list[str]:
     def _start(self) -> None:
         log_level = "DEBUG" if self._debug else "INFO"
         cmd = [
-            "python3",
-            "-s",
-            "-m",
-            "jupyter",
-            "labhub",
+            "jupyterhub-singleuser",
             "--ip=0.0.0.0",
             "--port=8888",
             "--no-browser",
             f"--notebook-dir={self._home!s}",
-            f"--hub-prefix={self._stash['jupyterhub_path']}",
-            f"--hub-host={self._stash['external_host']}",
```

Comment:
I gather we don't need these flags any more? What were they for and why don't we need them?

Reply:
Part of jupyter-labhub.